### ISSUE TITLE
oelint.var.licenseremotefile FP when using variable from recipe

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#141]
False positive reported for
```
LIC_FILES_CHKSUM = "file://${A}/LICENSE;md5=sjdjasdjhddh"
A = "${S}"
```
